x/payment/module.go

```diff
@@ -126,7 +124,7 @@ func (AppModule) QuerierRoute() string { return types.QuerierRoute }
 
 // LegacyQuerierHandler returns the capability module's Querier.
 func (am AppModule) LegacyQuerierHandler(legacyQuerierCdc *codec.LegacyAmino) sdk.Querier {
-	return keeper.NewQuerier(am.keeper, legacyQuerierCdc)
+	return nil
```

Comment:
I'm a little bit confused 😅  why nil is used in return

Reply:
Probably just to fulfill some legacy interface. We might be able to delete some of this stuff. should be investigated in #57